src/plugins/crypto/gpg.c

```diff
@@ -841,6 +863,10 @@ int CRYPTO_PLUGIN_FUNCTION (gpgCall) (KeySet * conf, Key * errorKey, Key * msgKe
 		ELEKTRA_SET_ERROR (ELEKTRA_ERROR_CRYPTO_GPG, errorKey, "GPG reported a bad signature");
 		break;
 
+	case 42:
+		// error has been set to errorKey by the child process
```

Comment:
Yes, but the child process has already exited, who will set the error here (in the parent process)? You need to communicate which error to set.

Reply:
But isn't `errorKey` a pointer pointing to the exact same address in both the child and the parent process?

Reply:
Oop sorry, I see where I went wrong.... I'll fix it right away.